pkg/provisioner/ironic/ironic.go

```diff
@@ -237,10 +237,15 @@ func (p *ironicProvisioner) ValidateManagementAccess(credentialsChanged bool) (r
 		ironicNode, err = nodes.Create(
 			p.client,
 			nodes.CreateOpts{
-				Driver:        p.bmcAccess.Driver(),
-				BootInterface: p.bmcAccess.BootInterface(),
-				Name:          p.host.Name,
-				DriverInfo:    driverInfo,
+				Driver:              p.bmcAccess.Driver(),
+				BootInterface:       p.bmcAccess.BootInterface(),
+				Name:                p.host.Name,
+				DriverInfo:          driverInfo,
+				InspectInterface:    p.bmcAccess.InspectInterface(),
```

Comment:
I'd be happy to just pass `"inspector"` here. At the moment the Ironic provisioner relies on being able to get all of the data provided by IPA from ironic-inspector. We'd have to completely change how the provisioner works before a bmc driver could produce valid results by customising the InspectInterface, so I think it's fine not to give them the option for now.